spec/system/case_contacts/new_spec.rb

```diff
@@ -177,6 +180,7 @@
 
         fill_in "case-contact-duration-hours", with: "1"
         fill_in "case-contact-duration-minutes", with: "45"
+        fill_in "Miles driven", with: "0"
```

Comment:
This should not be needed- empty field should not make save invalid, that would be bad user experience. I think the right solution here is to make the controller smart enough to translate `''` to "no value" / nil / 0 and not try to set the value.